cms/static/js/models/xblock_info.js

```diff
@@ -186,10 +190,50 @@ function(Backbone, _, str, ModuleUtils) {
         },
 
         /**
+         * Return true if subsection content is required to show.
+         * currently, check subsection for an entrance exam.
+        */
+        showSubSectionContent: function() {
```

Comment:
This is a very confusing method because the subsection content (ie, units) are still shown. What is being hidden in the release date, delete icon, etc.

Again, is_entrance_exam_subsection should not be in this file, so the xblock needs to state somehow that it does not want subsection control actions (or some similar name-- I haven't given it much thought).